src/screens/OMPSwitch/OMPSwitchHelper.res

```diff
@@ -24,7 +24,7 @@ module ListBaseComp = {
 
 module AddNewMerchantProfileButton = {
   @react.component
-  let make = (~user, ~setShowModal, ~customPadding="", ~customStyle="", ~customHRTagStyle="") => {
+  let make = (~user, ~setShowModal, ~customPadding="", ~customStyle="", ~customHRTagStyle="",~addProfileBtnStyle="") => {
```

Comment:
can you rename this variable to more generic name as this component is being used for merchant also ?

Reply:
Updated the variable name.